### PR TITLE
assorted: add russian title filters

### PR DESCRIPTION
**Pull request by spirkaa**

### CODE REVIEW

File: src/Jackett.Common/Definitions/riperam.yml [hunk 1]
```diff
@@ -894,6 +901,11 @@ search:
     st: 0
     ch: 300
     t: 0
+  keywordsfilters:
+    - name: re_replace # S01 to 1
+      args: ["(?i)\\bS0*(\\d+)\\b", "$1"]
+    - name: re_replace # S01E01 to 1 1
+      args: ["(?i)\\bS0*(\\d+)E0*(\\d+)\\b", "$1 $2"]
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/riperam.yml:908]: RiperAM ignores keywords which are fewer than 3 characters, so season/episode numbers on their own would just be ignored, and only the series name would be searched for. You could add an `andmatch` filter to the row selector and that should semi-resolve the issue.

File: src/Jackett.Common/Definitions/piratbitl.yml [hunk 2]
```diff
@@ -720,15 +725,37 @@ search:
...
+          args: ["Пифагор", "Pifagor"]
+        - name: replace
+          args: ["Невафильм", "Nevafilm"]
+        - name: replace
+          args: ["Лицензия", "Lic"]
+        - name: replace
+          args: ["селезень", "selezen"]
         # normalize to SXXEYY format
         - name: re_replace
-          args: ["[\\:\\-\\/\\|]", " "]
```
Comment (ilike2burnthing) [inline src/Jackett.Common/Definitions/piratbitl.yml:725]: Did you mean to remove this completely and only now replace ` / `? (same for piratbit obviously)

File: src/Jackett.Common/Definitions/riperam.yml (hunk 1 above)
Comment (spirkaa) [reply src/Jackett.Common/Definitions/riperam.yml:908]: This is more just a quick hack to allow series search from sonarr.

Comment (ilike2burnthing) [reply src/Jackett.Common/Definitions/riperam.yml:908]: Yea, that's fine, but for those searching manually, the `andmatch` filter will be useful.

**Review by ilike2burnthing — APPROVED** [review]

**Review by garfield69 — APPROVED** [review]

**Comment by garfield69** [comment]
Thank you.

**Comment by garfield69** [comment]
v0.20.892

**Pull request merged**